smithy-model/src/main/java/software/amazon/smithy/model/loader/ModelAssembler.java

```diff
@@ -142,7 +157,6 @@ public ModelAssembler reset() {
         validators.clear();
         suppressions.clear();
         documentNodes.clear();
-        properties.clear();
```

Comment:
These properties should have never of been cleared during a call to reset(). Reset should clear out transient state of the assembler, not configuration like properties.